src/toil/batchSystems/abstractBatchSystem.py

```diff
@@ -280,16 +280,6 @@ def setEnv(self, name, value=None):
                 raise RuntimeError("%s does not exist in current environment", name)
         self.environment[name] = value
 
-    def _getResultsFileName(self, toilPath):
-        """
-        Get a path for the batch systems to store results. GridEngine, slurm,
-        and LSF currently use this and only work if locator is file.
```

Comment:
Totally unused, so I removed it.

Reply:
Fantastic.  I too cannot find its use anywhere.